packages/dialtone-vue3/recipes/leftbar/style/leftbar_row.less

```diff
@@ -81,6 +81,21 @@
     }
   }
 
+  &__unread-count-badge {
+    border-top-right-radius: var(--dt-size-radius-0);
+    border-bottom-right-radius: var(--dt-size-radius-0);
+  }
+
+  &__unread-mention-count-badge {
+    border-top-left-radius: var(--dt-size-radius-0);
+    border-bottom-left-radius: var(--dt-size-radius-0);
+    background-color: var(--dt-color-purple-500);
+  }
+
+  &__unread-mention-count-only-badge {
+    background-color: var(--dt-color-purple-500);
+  }
```

Comment:
add `color: var(--dt-color-foreground-primary-inverted`

Have to account for Dark Mode. We'll probably have to arrive at a Theme category Design Token to account for this new addition. For now, this will do.

| Before | After |
|--------|--------|
| <img width="294" alt="image" src="https://github.com/dialpad/dialtone/assets/1165933/4e2ac966-5770-46d3-afea-948a1ddefb0a"> | <img width="287" alt="image" src="https://github.com/dialpad/dialtone/assets/1165933/3572c3b9-6ee7-4748-b931-46cff7642edd"> |

Reply:
Will do! Thanks!